### PR TITLE
sanitizing message from ExceptionTelemetry

### PR DESCRIPTION
**Pull request by eddynaka**
Fix Issue #546 .
<Short description of the fix.>

- [X] I ran Unit Tests locally.
- [X] CHANGELOG.md updated with one line description of the fix, and a link to the original issue if available.

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public surface reviewed
- [ ] CHANGELOG.md updated with one line description of the fix, and a link to the original issue.

The PR will trigger build, unit tests, and functional tests automatically. Please follow [these](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/.github/CONTRIBUTING.md) instructions to build and test locally.

Notes for reviewers:

- We support [comment build triggers](https://docs.microsoft.com/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers)
  - `/AzurePipelines run` will queue all builds
  - `/AzurePipelines run <pipeline-name>` will queue a specific build

### CODE REVIEW

File: BASE/src/Microsoft.ApplicationInsights/DataContracts/ExceptionTelemetry.cs [hunk 1]
```diff
@@ -340,6 +340,7 @@ public void SetParsedStack(System.Diagnostics.StackFrame[] frames)
         void ITelemetry.Sanitize()
         {
             // Sanitize on the ExceptionDetails stack information for raw stack and parsed stack is done while creating the object in ExceptionConverter.cs
+            this.Message = this.Message.SanitizeMessage();
```
Comment (cijothomas) [inline BASE/src/Microsoft.ApplicationInsights/DataContracts/ExceptionTelemetry.cs]: please add a unit test for the same.

**Comment by eddynaka** [comment]
> This PR still needs an entry in the Changelog

just updated! :)

**Pull request merged**